tests/integration/components/test_radio_open.py

```diff
@@ -0,0 +1,10 @@
+from tests.integration.integration_test_case import IntegrationTestCase
+
+
+class TestOpenRadio(IntegrationTestCase):
+    def setUp(self):
+        super().setUp()
+        self.launchSurvey("test_radio_open")
+
+    def test_label_appears(self):
+        self.assertInBody("Enter your favourite drink")
```

Comment:
I don't think this test is necessary, as it's duplicating the behaviour in 'When I view a write-in radio and the open option is set to true, detail answer label should be displayed' functional test.

Reply:
If we remove this test we won't have "required test coverage of 100%" and travis will fail.

Reply:
Ok, but you shouldn't need to create an integration test to test the jinja filter changes?

Reply:
I changed it and moved into jinja filters unit tests file